pkg/osbuild/kickstart_stage.go

```diff
@@ -218,3 +230,26 @@ func NewKickstartStageOptionsWithLiveIMG(
 
 	return options, nil
 }
+
+// IncludeRaw is used for adding raw text as an extension to the kickstart
+// file. First it changes the filename of the existing kickstart stage options
+// and then creates a new file with the given raw content and an %include
+// statement at the top that points to the renamed file. The new raw content is
+// generated in place of the original file and is returned as an fsnode.File.
+// The raw content *should not* contain the %include statement.
+func (options *KickstartStageOptions) IncludeRaw(raw string) (*fsnode.File, error) {
```

Comment:
This is clever! It also makes me slightly sad that we need workarounds like this but I guess it's unavoidable